jsonschema-hyperschema.xml

```diff
@@ -863,81 +865,30 @@ GET /foo/
                 </section>
             </section>
 
-            <section title="Submission Form Properties">
+            <section title="submissionEncType" anchor="submissionEncType">
                 <t>
-                    The following properties also apply to Link Description Objects, and provide functionality analogous to <xref target="W3C.CR-html5-20140731">HTML forms</xref>, by providing a means for making a request with client- or user-selected information.
```

Comment:
Mixed feelings about removing references to the biggest hypermedia format, but eh. Maybe it's for the greater good.